app/components/index/template.hbs

```diff
@@ -2,24 +2,24 @@
   <h3>Computerised Maintenance Management System</h3>
   <div class='d-flex m-5'>
     <button
-      class='btn col-5 mx-auto btn-danger'
+      class='btn-main btn col-5 mx-auto btn-danger'
       type='button'
       {{on 'click' this.onRequest}}
     >Open requests</button>
     <button
-      class='btn col-5 mx-auto btn-success'
+      class='btn-main btn col-5 mx-auto btn-success'
       type='button'
       {{on 'click' this.onKpi}}
```

Comment:
Maybe try to name actions by it's functionality because it might be hard to maintain code which isn't obvious while reading. When you see onKpi you don't know if you ar showing KPI, deleting it or even creating. Same rule for all actions